1.0.39版本在国产uos系统环境下存在GLIBC版本过高问题

您好，最新版本在国产uos系统下（仍然是麒麟9000C cpu），存在GLIBC版本过高的问题，Netcatty-1.0.39-linux-arm64.AppImage无法运行。
安装Netcatty-1.0.39-linux-arm64.deb 文件运行报错及测试如下：

uos@uos-PC:~Downloads$ sudo dpkg --install Netcatty-1.0.39-linux-arm64.deb 
正在选中未选择的软件包 netcatty。
(正在读取数据库 ... 系统当前共安装有 295390 个文件和目录。)
准备解压 Netcatty-1.0.39-linux-arm64.deb  ...
正在解压 netcatty (1.0.39) ...
/home/uos/Downloads/Netcatty-1.0.39-linux-arm64.deb
正在设置 netcatty (1.0.39) ...
update-alternatives: 使用 /opt/Netcatty/netcatty 来在自动模式中提供 /usr/bin/netcatty (netcatty)
正在处理用于 desktop-file-utils (0.23-4+sign) 的触发器 ...
正在处理用于 mime-support (3.62) 的触发器 ...
正在处理用于 bamfdaemon (0.5.4.2-1) 的触发器 ...
Rebuilding /usr/share/applications/bamf-2.index...
正在处理用于 hicolor-icon-theme (0.17-2+sign) 的触发器 ...
uos@uos-PC:~Downloads$ netcat
netcat    netcatty  
uos@uos-PC:~/Downloads$ netcatty 
innerError Error: Cannot find module '../build/Debug/pty.node'
Require stack:
- /opt/Netcatty/resources/app.asar/node_modules/node-pty/lib/unixTerminal.js
- /opt/Netcatty/resources/app.asar/node_modules/node-pty/lib/index.js
- /opt/Netcatty/resources/app.asar/electron/bridges/terminalBridge.cjs
- /opt/Netcatty/resources/app.asar/electron/main.cjs
- 
    at Module._resolveFilename (node:internal/modules/cjs/loader:1428:15)
    at s._resolveFilename (node:electron/js2c/browser_init:2:134237)
    at defaultResolveImpl (node:internal/modules/cjs/loader:1066:19)
    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1071:22)
    at Module._load (node:internal/modules/cjs/loader:1234:37)
    at c._load (node:electron/js2c/node_init:2:17999)
    at TracingChannel.traceSync (node:diagnostics_channel:328:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:252:24)
    at Module.require (node:internal/modules/cjs/loader:1511:12)
    at require (node:internal/modules/helpers:152:16) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/opt/Netcatty/resources/app.asar/node_modules/node-pty/lib/unixTerminal.js',
    '/opt/Netcatty/resources/app.asar/node_modules/node-pty/lib/index.js',
    '/opt/Netcatty/resources/app.asar/electron/bridges/terminalBridge.cjs',
    '/opt/Netcatty/resources/app.asar/electron/main.cjs',
    undefined
  ]
}
Uncaught exception: Error: /lib/aarch64-linux-gnu/libc.so.6: version **`GLIBC_2.34' not found** (required by /opt/Netcatty/resources/app.asar.unpacked/node_modules/node-pty/build/Release/pty.node)
    at process.func [as dlopen] (node:electron/js2c/node_init:2:2617)
    at Module._extensions..node (node:internal/modules/cjs/loader:1929:18)
    at Object.func [as .node] (node:electron/js2c/node_init:2:2844)
    at Module.load (node:internal/modules/cjs/loader:1488:32)
    at Module._load (node:internal/modules/cjs/loader:1307:12)
    at c._load (node:electron/js2c/node_init:2:17999)
    at TracingChannel.traceSync (node:diagnostics_channel:328:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:252:24)
    at Module.require (node:internal/modules/cjs/loader:1511:12)
    at require (node:internal/modules/helpers:152:16) {
  code: 'ERR_DLOPEN_FAILED'
}
/opt/Netcatty/resources/app.asar/electron/main.cjs:28
  throw err;
  ^

Error: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /opt/Netcatty/resources/app.asar.unpacked/node_modules/node-pty/build/Release/pty.node)
    at process.func [as dlopen] (node:electron/js2c/node_init:2:2617)
    at Module._extensions..node (node:internal/modules/cjs/loader:1929:18)
    at Object.func [as .node] (node:electron/js2c/node_init:2:2844)
    at Module.load (node:internal/modules/cjs/loader:1488:32)
    at Module._load (node:internal/modules/cjs/loader:1307:12)
    at c._load (node:electron/js2c/node_init:2:17999)
    at TracingChannel.traceSync (node:diagnostics_channel:328:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:252:24)
    at Module.require (node:internal/modules/cjs/loader:1511:12)
    at require (node:internal/modules/helpers:152:16) {
  code: 'ERR_DLOPEN_FAILED'
}

Node.js v24.11.1
uos@uos-PC:~/Downloads$ 
uos@uos-PC:~/Downloads$  ldd --version
**ldd (Debian GLIBC 2.28.31-deepin1) 2.28**
Copyright (C) 2018 自由软件基金会。
这是一个自由软件；请见源代码的授权条款。本软件不含任何没有担保；甚至不保证适销性
或者适合某些特殊目的。
由 Roland McGrath 和 Ulrich Drepper 编写。

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -79,13 +79,14 @@ jobs:
             release/*.tar.gz
           if-no-files-found: ignore
 
-  # Dedicated job for Linux ARM64 — builds inside Debian Buster (GLIBC 2.28)
-  # to ensure compatibility with older distros like UOS/Deepin.
+  # Dedicated job for Linux ARM64 — builds inside Debian Bullseye (GLIBC 2.31)
+  # to ensure compatibility with older distros like UOS/Deepin (GLIBC 2.28).
+  # Key: GLIBC < 2.34 avoids the libpthread-merge symbol requirement.
   build-linux-arm64:
     name: build-linux-arm64
     runs-on: ubuntu-24.04-arm
     container:
-      image: debian:buster
+      image: debian:bullseye
     env:
       VITE_SYNC_GITHUB_CLIENT_ID: ${{ secrets.VITE_SYNC_GITHUB_CLIENT_ID }}
       VITE_SYNC_GOOGLE_CLIENT_ID: ${{ secrets.VITE_SYNC_GOOGLE_CLIENT_ID }}
@@ -94,10 +95,6 @@ jobs:
     steps:
       - name: Install build dependencies
         run: |
-          # Buster is EOL — switch to archived repos
-          sed -i 's|deb.debian.org/debian|archive.debian.org/debian|g' /etc/apt/sources.list
-          sed -i 's|deb.debian.org/debian-security|archive.debian.org/debian-security|g' /etc/apt/sources.list
-          sed -i '/buster-updates/d' /etc/apt/sources.list
           apt-get update
           apt-get install -y curl build-essential python3 git libfuse2 file rpm
           curl -fsSL https://deb.nodesource.com/setup_20.x | bash -

